media-libs/openpgl/openpgl-0.5.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Intel Open Path Guiding Library"
+HOMEPAGE="https://github.com/OpenPathGuidingLibrary/openpgl"
+SRC_URI="https://github.com/OpenPathGuidingLibrary/openpgl/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+X86_CPU_FLAGS=( sse4_2:sse4_2 avx2:avx2 avx512dq:avx512dq )
+CPU_FLAGS=( ${X86_CPU_FLAGS[@]/#/cpu_flags_x86_} )
+IUSE="${CPU_FLAGS[@]%:*}"
+SLOT="0"
+
+RDEPEND="
+	media-libs/embree
+	dev-cpp/tbb:=
+"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DOPENPGL_ISA_AVX2=$(usex cpu_flags_x86_avx2)
+		-DOPENPGL_ISA_AVX512=$(usex cpu_flags_x86_avx512dq)
+		-DOPENPGL_ISA_SSE4=$(usex cpu_flags_x86_sse4_2)
+	)
+
+	# Disable asserts
+	append-cppflags -DNDEBUG
```

Comment:
Why are you disabling them unconditionally?

Reply:
I guess you want me to add a `debug` useflag that toggles this?
Otherwise it makes no sense to have the debug asserts on when building a non debug build.

Reply:
Yes, I think that's a reasonable approach.